component/encoder/v1/encoder.pb.gw.go

```diff
@@ -15,7 +15,7 @@ import (
 
 	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
 	"github.com/grpc-ecosystem/grpc-gateway/v2/utilities"
-	"go.viam.com/api/common/v1"
```

Comment:
@JohnN193 or @jeremyrhyde I know we discussed why these unrelated files get changed when we change slam.proto in https://github.com/viamrobotics/api/pull/231/files#diff-2170dff6c9599d02586a135ae17ca0a4d9e2df172c7ab24ef4a04829aad8a101. Do you remember why and mind reminding me?